### PR TITLE
9369 closed schools

### PR DESCRIPTION
**Pull request by rianfowler**
Summary of UI updates can be found [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9369#issuecomment-382548507): 

screen shot 2018-06-26 at 16 37 38
screen shot 2018-06-26 at 16 37 23
screen shot 2018-06-26 at 16 36 57
screen shot 2018-06-26 at 16 36 46

### CODE REVIEW

File: src/applications/gi/sass/partials/_gi-search-page.scss [hunk 1]
```diff
@@ -148,5 +152,20 @@
       }
     }
 
+    .caution-flag-new {
```
Comment (rianfowler) [inline src/applications/gi/sass/partials/_gi-search-page.scss]: new styles are behind a feature flag

**Comment by rianfowler** [comment]
Updated the comment with new screen shot of the changes. This data is on dev which locally run `vets-website` should point to. 

Single warning detail: http://localhost:3001/gi-bill-comparison-tool/profile/15002244
Single warning search: http://localhost:3001/gi-bill-comparison-tool/search?name=utah+state
Double warning search: http://localhost:3001/gi-bill-comparison-tool/search?name=neumont
Double warning detail: http://localhost:3001/gi-bill-comparison-tool/profile/21000944

File: src/applications/gi/components/profile/HeadingSummary.jsx [hunk 2]
```diff
@@ -35,9 +35,15 @@ class HeadingSummary extends React.Component {
       <div className="heading row">
         <div className="usa-width-two-thirds medium-8 small-12 column">
           <h1>{it.name}</h1>
+          {__BUILDTYPE__ !== 'production' && (<AlertBox
+            content={(<p>Are you enrolled in this school? <a href="https://www.benefits.va.gov/GIBILL/FGIB/Restoration.asp" onClick={this.props.onViewWarnings}>Find out how the VA can restore your benefits</a></p>)}
```
Comment (jbalboni) [inline src/applications/gi/components/profile/HeadingSummary.jsx]: It seems odd that this is a link with both an onClick and an external href. It also may need to open in a new tab.

**Review by jbalboni — APPROVED** [review]
One question, but LGTM otherwise.

**Pull request merged**